aws-lambda-function/variables.tf

```diff
@@ -1,110 +1,126 @@
 
-variable service {
+variable "service" {
   type        = string
   description = "Service for tagging and naming. See [doc](../README.md#consistent-tagging)"
 }
 
-variable owner {
+variable "owner" {
   type        = string
   description = "Owner for tagging and naming. See [doc](../README.md#consistent-tagging)"
 }
 
-variable source_s3_bucket {
+variable "source_s3_bucket" {
   type        = string
   description = "Bucket holding lambda source code."
   default     = null
 }
 
-variable source_s3_key {
+variable "source_s3_key" {
   type        = string
   description = "Key identifying location of code."
   default     = null
 }
 
-variable handler {
+variable "handler" {
   type        = string
   description = "Name of the lambda handler."
 }
 
-variable runtime {
+variable "runtime" {
   type        = string
   description = "Lambda language runtime."
 }
 
-variable timeout {
+variable "timeout" {
   type        = number
   description = "Execution timeout for the lambda."
   default     = null
 }
 
-variable environment {
+variable "environment" {
   type        = map(string)
   description = "Map of environment variables."
   default     = {}
 }
 
-variable kms_key_arn {
+variable "kms_key_arn" {
   type        = string
   description = "KMS key used to encrypt environment variables."
   default     = null
 }
 
-variable source_code_hash {
+variable "source_code_hash" {
   type    = string
   default = null
 }
 
-variable filename {
+variable "filename" {
   type    = string
   default = null
 }
 
-variable log_retention_in_days {
+variable "log_retention_in_days" {
   type    = number
   default = null
 }
 
-variable function_name {
+variable "function_name" {
   type        = string
   description = "If not set, function use default naming convention of $project-$env-$service. See local.name in main.tf"
   default     = null
 }
 
-variable function_description {
+variable "function_description" {
   type        = string
   description = "Description for lambda function."
   default     = ""
 }
 
-variable publish_lambda {
+variable "publish_lambda" {
   type        = bool
   description = "Whether to publish creation/change as new lambda function version."
   default     = false
 }
 
-variable lambda_role_path {
+variable "lambda_role_path" {
   type        = string
   description = "The path to the IAM role for lambda."
   default     = null
 }
 
-variable at_edge {
+variable "at_edge" {
   type        = bool
   description = "Is this lambda going to be used with a Cloufront distribution? If you set this, you will not have control over log retention, and you cannot include environment variables."
   default     = false
 }
 
-variable reserved_concurrent_executions {
+variable "reserved_concurrent_executions" {
   type        = number
   description = "Set reserved_concurrent_executions for this function. See [docs](https://docs.aws.amazon.com/lambda/latest/dg/configuration-concurrency.html)."
   default     = -1 // aws default
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Single line comments should begin with #